cluster-sync/sync.sh

```diff
@@ -79,6 +80,40 @@ function wait_cdi_available {
   fi
 }
 
+OLD_CDI_VER_PODS="/tmp/old_cdi_ver_pods"
```

Comment:
maybe `./_out/tests` is better than `/tmp`?

Reply:
sure